api/lib/domain/usecases/create-certification-center-membership-for-sco-organization-member.js

```diff
@@ -6,26 +8,34 @@ const createCertificationCenterMembershipForScoOrganizationMember = async functi
 }) {
   const existingMembership = await membershipRepository.get(membership.id);
 
-  if (membership.isAdmin && existingMembership.organization.isScoAndHasExternalId) {
-    const existingCertificationCenter = await certificationCenterRepository.findByExternalId({
-      externalId: existingMembership.organization.externalId,
+  if (!membership.isAdmin || !existingMembership.organization.isScoAndHasExternalId) return;
```

Comment:
Juste une toute petite remarque, le titre du 1er commit gagnerait à être modifié pour qu'il soit plus clair que le comportement du usecase n'est pas modifié et que seul le code flow est amélioré. Aussi je suggère un renommage du titre du commit dans le style : 
_refactor(api): modify code execution flow_ → _refactor(api): improve code execution flow for clarity_